student,q11,q11-failed,q31,q31-failed,q211,q211-failed,q21,q21-failed,total_grade,comments
test,0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/envs/carme/lib/python3.6/doctest.py"", line 1330, in __run', '        compileflags, 1), test.globs)']",0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/envs/carme/lib/python3.6/doctest.py"", line 1330, in __run', '        compileflags, 1), test.globs)']",0.0,"['Expecting:', ""    \\'repeekkoob\\'"", '**********************************************************************', 'Line 8, in /Users/jasonkuruzovich/githubdesktop/0_class/summer_2019/notebooks/tests/q211.py 0']",0.0,"['Failed example:', '    woman_asking', 'Exception raised:', '    Traceback (most recent call last):']",0.0,
test,0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/envs/carme/lib/python3.6/doctest.py"", line 1330, in __run', '        compileflags, 1), test.globs)']",0.0,"['Exception raised:', '    Traceback (most recent call last):', '      File ""/anaconda3/envs/carme/lib/python3.6/doctest.py"", line 1330, in __run', '        compileflags, 1), test.globs)']",0.0,"['Expecting:', ""    \\'repeekkoob\\'"", '**********************************************************************', 'Line 8, in /Users/jasonkuruzovich/githubdesktop/0_class/summer_2019/notebooks/tests/q211.py 0']",0.0,"['Failed example:', '    woman_asking', 'Exception raised:', '    Traceback (most recent call last):']",0.0,
